.vimrc

```diff
@@ -420,6 +434,8 @@ hi illuminatedWord cterm=underline gui=underline
 " ============================================================================
 " BETTER COMMENTS 
 " ============================================================================
+let g:bettercomments_skipped = ['python']
```

Comment:
I need to review this change @radyz  probably is not need it now. I think that I added it because it breaks formatter or the code highlighting